### DISCUSSION TITLE
Where to build the docker image

### DISCUSSION THREAD

**Discussion by rose-jinyang** [AxisCommunications/acap-computer-vision-sdk-examples#76] [Q&A] [answered]
Hello
How are you?
Thanks for contributing to this project.
I have a quick question.
Which device & platform should the app docker image be built on?
I can NOT see this in the README.
I think it will be a PC platform(eg: Windows 10 or Ubuntu 20.04), right?

**Comment by Corallo** [comment, marked as answer]
Hello @rose-jinyang 
Yes, we recommend building the docker image on your desktop, you can do it on any operative system.

**Reply by rose-jinyang** [reply]
Hello
I tried to run the sample "minimal-ml-inference" on my AXIS camera device.
I followed the below steps as README

export ARCH=armv7hf
export CHIP=tpu

export AXIS_TARGET_IP=x.x.x.x
export DOCKER_PORT=2376
docker --tlsverify -H tcp://$AXIS_TARGET_IP:$DOCKER_PORT system prune -af

But I met the following issue
unable to resolve docker endpoint: open /home/osboxes/.docker/ca.pem: no such file or directory

Where and how should I generate ca.pem file?

**Reply by Corallo** [reply]
Hi, if you want to use tlsverify you should set up [tls first](https://github.com/AxisCommunications/docker-acap#securing-the-docker-acap-using-tls).
Otherwise, you can change the port from 2376 to 2375, remove the --tlsverify parameter 
and in the camera UI go on the docker acap [...] -> Setting -> Use TLS -> No

We do recommend to keep using tls for security reason.

**Reply by rose-jinyang** [reply]
How can I go to the docker acap [...] -> Setting -> Use TLS in the camera UI.
I can NOT find it in web GUI

![image](https://user-images.githubusercontent.com/47862419/185178165-ee9dd1e5-e741-483d-bad2-27a0bed77c1e.png)

**Reply by Corallo** [reply]
You have first to install the [docker-acap](https://github.com/AxisCommunications/docker-acap)
